ga1_draft.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import division
+import numpy as np  
+import matplotlib.pyplot as plt
+from pymatgen import MPRester 
+
+# import/truncate raw data (volume, density, energies, etc)
+	m = MPRester("ID")
```

Comment:
Definitely, @tschaume. I just wanted to go through the mechanics of a pull request (PR), merger, and re-syncing a fork with upstream. @HGeerlings had this scaffold at the ready, so we walked through the process.